src/main/resources/network/brightspots/rcv/hints_cvr_files.txt

```diff
@@ -7,8 +7,12 @@ For each of your CVR files, provide the necessary information and then use the A
 Provider (required): The vendor/machine that generated (produced) the file. After you select the field, the tabulator will fill in as many of the other fields as it can based on what it knows about that provider. You can adjust those values as necessary.
 
 Path (required): Location of the CVR file.
-  * Example: /Users/test data/2015-portland-mayor-cvr.xlsx
-  * Example: C:\Users\test data\precinct14.json
+  * Example: /test data/2015-portland-mayor-cvr.xlsx
+  * Example: C:\test data\2015-portland-mayor-cvr.xlsx
+NOTE: You can select multiple files by holding down Shift in the dialog. You can also manually add multiple if you enter the values in the Path input box, separated with semicolons:
```

Comment:
This guidance is Windows-specific. On macOS, you can select a set of consecutive files by Shift-clicking, or you can select two or more non-adjacent files by Command-clicking.

Reply:
Updated this, thanks!